pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name:
+
+`ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```

Comment:
I'm not really sure if we should include the `grep` and `awk` examples here. In general the `ansible-galaxy` page should show how the command works. If someone wants to filter the role id they can use any tools they want.

Reply:
I agree with @marchersimon on this one. we generally want to avoid demonstrating the functionality of other commands here.

Reply:
@papanito you can remove the `grep` and `awk` parts and explain what the `info` subcommand displays here.
```suggestion
`ansible-galaxy info {{username}}.{{role_name}}
```
Also, if the `.` is mandatory I think we should use the same token syntax with the first 2 examples.

Reply:
@navarroaxel I suggested that above, I didn't see that the first two examples had it in the token syntax.
But why should it not be outside? We're doing that in all pages where two arguments are separated by a single character.

Reply:
@marchersimon It's like the `ssh.md`: the `@` is outside `{{username}}@{{host}}` because it's mandatory and the `@` is not a placeholder, the same apply here.

Reply:
I assume then the first two examples should also be updated?